src/main/java/Task.java

```diff
@@ -0,0 +1,46 @@
+public class Task {
```

Comment:
Javadocs is not present in Class and Methods.

```java
/**
 * Returns lateral location of the specified position.
 * If the position is unset, NaN is returned.
 *
 * @param x X coordinate of position.
 * @param y Y coordinate of position.
 * @param zone Zone of position.
 * @return Lateral location.
 * @throws IllegalArgumentException  If zone is <= 0.
 */
public double computeLocation(double x, double y, int zone)
        throws IllegalArgumentException {
    //...
}
```

Reply:
Thanks for the feedback, have added JavaDoc for roughly half of the non-private methods. Will continue working towards 100% JavaDoc coverage!